l3packages/xfp/testfiles/xfp000.tlg

```diff
@@ -3,10 +3,11 @@ Don't change this file in any respect.
 Author: Bruno Le Floch
 (xfp.sty (xparse.sty
 Package: xparse ....-..-.. L3 Experimental document command parser
```

Comment:
maybe time to change the package description? and drop "Experimental"? no?

Reply:
Some further thought on file names: should we go for l3parse and only retain xparse.sty as an historical exception?

Reply:
@FrankMittelbach september last year we thought about `l3x..` for (user) packages and `l3somethingelse` for kernel stuff. so probably either `l3parse` or `l3xparse` depending on what this is now ...

Reply:
but we decided back that that we wouldn't change the name xparse because it is so widely used. My point of file names was more for the internal names in that case that get loaded by the kernel.

Reply:
so l3parse.sty here.  Sounds good to me.